domain/transfer/service.go

```diff
@@ -59,24 +62,28 @@ func (s service) CreateNegotiation(ctx context.Context, customerID, transferID,
 	if err != nil || customer.Did == nil {
 		return nil, err
 	}
+
 	var (
 		negotiation          *domain.TransferNegotiation
 		notificationEndpoint string
 	)
+
 	_, err = s.transferRepo.Update(ctx, customerID, transferID, func(transfer domain.Transfer) (*domain.Transfer, error) {
 		// Validate transfer
 		if transfer.Status == domain.TransferStatusCancelled || transfer.Status == domain.TransferStatusCompleted || transfer.Status == domain.TransferStatusAssigned {
 			return nil, errors.New("can't start new transfer negotiation when status is 'cancelled', 'assigned' or 'completed'")
 		}
+
 		// Create negotiation and share it to the other party
 		// TODO: Share transaction to this repository call as well
 		var err error
+
 		taskProperties := domain.TaskProperties{
 			RequesterID: *customer.Did,
 			OwnerID:     organizationDID,
 		}
 
-		// Pre-emptively resolve the receiver organization's notification endpoint to reduce clutter, avoiding to make FHIR tasks when the receiving party eOverdracht registration is faulty.
+		// Pre-emptively resolve the receiver organization's endpoints to reduce clutter, avoiding to make FHIR tasks when the receiving party eOverdracht registration is faulty.
```

Comment:
Although I don't really care of the tiny change, we _do_ specifically resolve the notification endpoint?